courses/pyladies/info.yml

```diff
@@ -18,6 +18,7 @@ plan:
   - lesson: beginners/cmdline
   - lesson: beginners/install
   - lesson: beginners/first-steps
+  - lesson: beginners/venv-setup
```

Comment:
Tohle je potřeba přidat u všech jednotlivých kurzů, kterých se to týká.

Reply:
rozumím. udělám to, pokud to lidi odsouhlasí.